molecule/delegated/tests/clevis/redhat.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+
+from ..util.util import get_ansible, get_variable, get_family_role_variable
+
+testinfra_runner, testinfra_hosts = get_ansible()
+
+
+def check_ansible_os_family(host):
+    if get_variable(host, "ansible_os_family", True) != "RedHat":
+        pytest.skip("ansible_os_family mismatch")
+
+
+def test_clevis_packages(host):
```

Comment:
``test_clevis_packages`` could be moved to main.py

Reply:
Done